big_tests/tests/shared_roster_SUITE.erl

```diff
@@ -143,7 +142,7 @@ add_user(Config) ->
 
 start_roster_module(ldap) ->
     case rpc(mim(), gen_mod, start_module,
-             [ct:get_config({hosts, mim, domain}), mod_shared_roster_ldap, get_ldap_args()]) of
+             [ct:get_config({hosts, mim, host_type}), mod_shared_roster_ldap, get_ldap_args()]) of
```

Comment:
Should be used `-import(domain_helper, [host_type/0]).`

Reply:
Yes, please do